addons/web/static/tests/webclient/commands/command_service_tests.js

```diff
@@ -551,11 +602,15 @@ QUnit.test("multi level commands with hotkey", async (assert) => {
     const otherNames = ["Cersei", "Lannister"];
     const action = () => ({
         placeHolder: "Who is the next King ?",
-        provide: () =>
-            otherNames.map((name) => ({
-                action: () => {},
-                name,
-            })),
+        providers: [
+            {
+                provide: () =>
+                    otherNames.map((name) => ({
+                        name,
+                        action: () => {},
+                    })),
+            },
```

Comment:
BTW, providers should be an array of functions, not an array of { provide: function }